library/java/net/openid/appauth/JsonUtil.java

```diff
@@ -497,4 +499,13 @@ String convert(String value) {
             return value;
         }
     }
+
+    public static String decodeBase64(String strEncoded) throws UnsupportedEncodingException {
```

Comment:
Okay I will move this method to Utils.java class which is better home for such methods.